packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -1,12 +1,6 @@
-import React, {
-  MutableRefObject,
-  ReactElement,
-  useEffect,
-  useRef,
-  useState,
-} from "react";
+import React, { ReactElement, useEffect, useState } from "react";
 import classnames from "classnames";
-import ReactDatePicker from "react-datepicker";
+import { ReactDatePicker } from "react-datepicker";
```

Comment:
That test failed because of this was changed to something that is not a component